docs/api.md

```diff
@@ -214,6 +214,31 @@ Always use `.done()` instead of `break`.
         rows.done();
     }
 
+### Rows.columns
+```javascript
+columns()
+```
+Call this if you need column names from the result of a select query.
+
+This method returns an array of objects, where each object has the following properties.
+
+| Property     | Value                                      |
+|--------------|--------------------------------------------|
+| `name`       | the result of `sqlite3_column_name`        |
+| `originName` | the result of `sqlite3_column_origin_name` |
+| `tableName`  | the result of `sqlite3_column_table_name`  |
+
+```javascript
```

Comment:
Two things here:
1. This file is auto-generated and your changes here will be discarded with re-generation (which will happen automatically on push to master)
2. I don't think we need the code example here, the documentation you provide in the documenting comment is completely sufficient, explaining what the returned objects will look like 😃

Reply:
(to fix this, simply run `deno --allow-read --allow-write docs/generate.js` and follow the instructions given)